workflows/diagnostics/fv3net/diagnostics/offline/compute.py

```diff
@@ -173,7 +173,9 @@ def _consolidate_dimensioned_data(ds):
     return ds_diagnostics, ds_scalar_metrics
 
 
-def _get_transect(ds_snapshot: xr.Dataset, grid: xr.Dataset, variables: Sequence[str]):
+def _get_transect(
+    ds_snapshot: xr.Dataset, grid: xr.Dataset, variables: Sequence[str], ptop: float,
```

Comment:
the added `ptop` argument is not used in this function.

Reply:
It actually needs to be passed to `interpolate_to_pressure_levels`, updated now